programs/cleos/main.cpp

```diff
@@ -1548,6 +1551,7 @@ int main( int argc, char** argv ) {
 
    bool verbose_errors = false;
    app.add_flag( "-v,--verbose", verbose_errors, localized("output verbose actions on error"));
+   app.add_flag("--print-request", print_request, localized("print HTTP request to STDERR"));
```

Comment:
Why STDERR and not STDOUT?

Reply:
No real reason for that. Just following a common practice - write real output to STDOUT and all diagnostic/debug message to STDERR. So, when you redirect output to file, you still see diagnostic messages in console and your file still contains valid output not mixed with the messages.

Reply:
ok